ntp.go

```diff
@@ -174,11 +258,21 @@ func QueryWithOptions(host string, opt QueryOptions) (*Response, error) {
 	if err != nil {
 		return nil, err
 	}
+	return parseTime(m, now)
+}
+
+func parseTime(m *msg, t4 ntpTime) (*Response, error) {
+	if m.OriginTime > t4 { // FIXME: Go 1.9 has monotonic clock
```

Comment:
What do you intend to do about this `FIXME`?

Reply:
I'm unsure what is the right thing to to _right now_. Is it possible to compile different code paths for Go 1.9+ and Go 1.8? I'm quite new to go lang.

Reply:
I've read [the paragraph on monotonic clock in Go](https://golang.org/pkg/time/#hdr-Monotonic_Clocks), seems, it's easier fix than I thought :)